bytesconv_table_gen.go

```diff
@@ -0,0 +1,149 @@
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"go/format"
+	"io/ioutil"
+	"log"
+)
+
+const (
+	toLower = 'a' - 'A'
+)
+
+func main() {
+	var hex2intTable = func() [256]byte {
```

Comment:
Can you change these to `:=` declarations like `hex2intTable := ...`. I prefer those, thanks!